docs/install/install_everest_helm_charts.md

```diff
@@ -54,6 +54,16 @@ Here are the steps to install Percona Everest and deploy additional database nam
         helm install everest-core percona/everest --namespace=everest-system --create-namespace --set pmm.enabled=true
         ```
 
+    ??? info "🔒 Install Percona Everest with TLS enabled"
+
+        Install Percona Everest with TLS enabled:
+
+        ```sh
+        helm install everest-core percona/everest \
+        --namespace everest-system \
+        --create-namespace
+        --set server.tls.enabled=true
+        ```
```

Comment:
Please attach a reference to the complete TLS documentation (something like 'See here for more info on configuring TLS')

Reply:
Done.